### DISCUSSION TITLE
Required hardware

### DISCUSSION THREAD

**Discussion by Repsionu** [leikoilja/ha-google-home#168] [Q&A]
It seems that lowly RPI 3+ with 1GB is not sufficent to build glocaltokens - out of memory :-(
So minimum HW for this addon?
Which version is required as pip install gets ver 0.25 not 0.30?
Best, JR

**Comment by KapJI** [comment]
Yeah, building `grpc` is a heavy task. I think this happens because of 4 jobs running in parallel. Try smaller number of jobs (1 or 2) by setting `GRPC_PYTHON_BUILD_EXT_COMPILER_JOBS` env var. It may take up to 1 hour to compile but eventually it should work.
